mne/channels/montage.py

```diff
@@ -308,6 +308,7 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         pos = apply_trans(neuromag_trans, pos)
 
     if ch_names is not None:
+        ch_names = set(ch_names)
```

Comment:
again this should be really inremental and it makes you adding a line while we don't win readability.